admin_auto_filters/filters.py

```diff
@@ -100,12 +103,15 @@ def has_output(self):
     def lookups(self, request, model_admin):
         return ()
 
+    def value(self):
+        return self.used_parameters.get(self.parameter_name).split(',') if self.used_parameters.get(self.parameter_name) else []
```

Comment:
Wouldn't this be better with classical if - else block?

Reply:
Sure I changed it.
Also had to add `var jQuery = django.jQuery, $ = jQuery;` to the beginning of the js file for it to work consistently.
To fix the bug of having to have an empty Media class in the Admin class, it probably would be better to attach the media to a form class surrounding the select as in [django-admin-rangefilter](https://github.com/silentsokolov/django-admin-rangefilter).
It does not seem correct to have a select outside of a form anyway.

Reply:
My recommendation would be

```python
parameter_name = self.used_parameters.get(self.parameter_name)
if parameter_name:
    return parameter_name.split(',')
return []
```